controllers/clusters/cassandra_controller.go

```diff
@@ -639,6 +639,8 @@ func (r *CassandraReconciler) handleUsersCreate(
 		return err
 	}
 
+	l.Info("User has been set on creation", "username", u.Name)
```

Comment:
User has been set on creation/deletion -> User has been added to the queue for creation/deletion